data/card_catalogue/unique-1.json

```diff
@@ -356,7 +356,7 @@
     "tier": "green",
     "details": {
       "rules": {
-        "active": "Burst into flames along with anyone you're touching. 3d6 damage to each of you.",
```

Comment:
I don't think we need to say it's going to hurt everyone, it's fire. The damage is appropriate for how crazy this effect clearly is